### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by jayvanpariya1679**

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+	                data-target="#navbarToggleExternalContent" aria-controls="navbarToggleExternalContent" 
+	                aria-expanded="false" aria-label="Toggle navigation"> 
+	                <i class="fa-solid fa-bars"></i></button> 
+                </section>
+            </div>
+            <div class="col-8"></div>
+            <div class="col-3" style="margin-top: 0.938rem"> 
+            	<a href="#email" style="margin-right: 0.75rem"><i class="fa-regular fa-envelope" style="color: #ffffff"></i></a> 
+            	<a href="#bell" style="margin-right: 0.75rem"><i class="fa-regular fa-bell" style="color: #ffffff"></i></a>
+            	<a href="#flag" style="margin-right: 0.75rem"><i class="fa-regular fa-flag" style="color: #ffffff"></i></a>
```
Comment (dix1999) [inline src/main/webapp/index.html]: Don't add direct style. Use class insted at everywhere. Like below.

.white-color {
    color: #ffffff
}

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+        <div class="container-fluid ">
+            <div class="row">
+                <div class="col-md mb-3">
+                    <div class="border bg-white"> <img alt="CPU TRAFFIC" src="images/setting.svg" style="width: 6.25rem; height: 6.25rem;"><span> CPU TRAFFIC </span><span>90%</span></div>
+                </div>
+                <div class="col-md mb-3">
+                    <div class="border bg-white"> <img alt="CPU TRAFFIC" src="images/google.svg" style="width: 6.25rem; height: 6.25rem;"> LIKES 41,410 </div>
+                </div>
+                <div class="col-md mb-3">
+                    <div class="border bg-white"> <img alt="CPU TRAFFIC" src="images/trolly.svg" style="width: 6.25rem; height: 6.25rem;"> SALES 760 </div>
```
Comment (dix1999) [inline src/main/webapp/index.html]: Don't add direct style. Create class insted and use it at everywhere.

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -0,0 +1,467 @@
...
+	                        <span class="badge bg-danger float-right" data-widget="collapse">8 new Members</span> 
+	                        </div>
+                        <div class="border-top">
+                            <div class="row mt-3">
+                                <div class="col-3"> <img class="direct-chat-img rounded-circle" style="width: 5rem; height: 5rem;" src="images/alexander.svg" alt="message user image">
+                                    <p align="center"> Alexander...<br>Today </p> 
+                                    <img class="direct-chat-img rounded-circle" style="width: 5rem; height: 5rem;" src="images/Alexander.svg" alt="message user image">
+                                    <p align="center"> Alexander<br>13 Jan </p>
+                                </div>
+                                <div class="col-3"> <img class="direct-chat-img rounded-circle" style="width: 5rem; height: 5rem" src="images/Norman.svg" alt="message user image">
```
Comment (dix1999) [inline src/main/webapp/index.html]: You have added a path of images but have not committed that.

File: src/main/webapp/index.html [hunk 4]
```diff
@@ -0,0 +1,467 @@
...
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/js/bootstrap.bundle.min.js"></script>
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" 
+    integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" 
+    crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js" 
+    integrity="sha384-cVKIPhGWiC2Al4u+LWgxfKTRIcfu0JTxR+EQDz/bgldoEyl4H0zUF0QKbrJ0EcQF" 
+    crossorigin="anonymous"></script>
+    <script>
+        var menuBtn = document.querySelector("#menu-btn")
```
Comment (dix1999) [inline src/main/webapp/index.html]: Why we need script to open and close side bar. Just remove it and use bootstrap insted.

File: src/main/webapp/style.css [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+.avatar {
+    vertical-align: middle;
+    width: 1.875rem; 
+    height: 1.875rem; 
+    border-radius: 50%;
+}
+
+body {
+    min-height: 100vh;
+    background-color: #fff;
```
Comment (dix1999) [inline src/main/webapp/style.css]: Create variable for color and use it here.

File: src/main/webapp/style.css [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    margin-left: 15.625rem;
+}
+
+#menu-btn {
+    background-color: var(black);
+    color: #fff;
+}
+
+#menu-btn:focus {
+    box-shadow: 0 0 0 0.15625rem #7952b344;
```
Comment (dix1999) [inline src/main/webapp/style.css]: Create variable for color and use it here.

**Review by dix1999 — CHANGES_REQUESTED** [review]

File: src/main/webapp/CSS/style.css [hunk 7]
```diff
@@ -0,0 +1,151 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+  --dropdown-gray: #6c757d;
```
Comment (dix1999) [inline src/main/webapp/CSS/style.css]: Let me give you an example. How to give color variable names. 

If we have variants of blue color then we can write like below.

    --blue: blue;
    --blue-primary: rgb(46 90 172);
    --blue-primary-1: rgb(0, 103, 177);
    --blue-light: rgb(238 242 250);

File: src/main/webapp/CSS/style.css [hunk 8]
```diff
@@ -0,0 +1,151 @@
...
+	
+}
+.direct-chat-messages{
+	max-height: 18.125rem;
+}
+.admin-heading{
+    margin-top: -0.5rem;
+    position: relative;
+    height: 3.313rem;
+    background-color: #3B71CA;
```
Comment (dix1999) [inline src/main/webapp/CSS/style.css]: Add color to variable.

File: src/main/webapp/index.html [hunk 9]
```diff
@@ -0,0 +1,713 @@
...
+                            <svg class="heart-svg" viewBox="0 0 256 256" xmlns="http://www.w3.org/2000/svg">
+                                <rect fill="none" height="256" width="256"></rect>
+                                <path d="M128,216S28,160,28,92A52,52,0,0,1,128,72h0A52,52,0,0,1,228,92C228,160,128,216,128,216Z" fill="none" stroke="#fff" stroke-linecap="round" stroke-linejoin="round" stroke-width="8"></path>
+                            </svg>
+                        </div>
+                        <div class="col-9 justify-content-center p-0 m-0 mt-1">
+                            <div class="info-box-content">
+                                <span class="info-box-text m-2">Mentions</span><br />
+                                <span class="info-box-number m-2"><strong>92,050</strong></span>
+                                <div class="progress bg-secondary mt-2 mb-2" style="height: 0.2rem;">
```
Comment (dix1999) [inline src/main/webapp/index.html:464]: Remove style from everywhere and create a custom class and use it.

**Review by dix1999 — CHANGES_REQUESTED** [review]

File: src/main/webapp/index.html (hunk 9 above)
Comment (jayvanpariya1679) [reply src/main/webapp/index.html:464]: progress bar can't support style with class

**Review by dix1999 — APPROVED** [review]

**Pull request merged**